R/drop_labels.R

```diff
@@ -17,7 +17,6 @@
 #'   skimr::skim()
 #' }
 drop_labels <- function(df) {
-  # browser()
```

Comment:
an old browser statement I found when I was deleting the ones I inserted.

Reply:
oops.  Thanks